src/native/corehost/hostpolicy/deps_resolver.cpp

```diff
@@ -581,12 +631,8 @@ bool deps_resolver_t::resolve_tpa_list(
         }
     }
 
-    // Convert the paths into a string and return it
-    for (const auto& item : items)
-    {
-        output->append(item.second.resolved_path);
-        output->push_back(PATH_SEPARATOR);
-    }
+    // Now that we have the complete list, store it in the host_runtime_contract and as a ; delimted string.
```

Comment:
We want to store TPA list as an array instead of a string with delimiter.

Reply:
From my understanding, we want to either stick to the string or use both to keep backwards compatibility.